core/string/translation.h

```diff
@@ -157,6 +157,12 @@ class TranslationServer : public Object {
 
 	void load_translations();
 
+#ifdef TOOLS_ENABLED
+	Vector<String> get_editor_locales() const;
+	void load_editor_tool_translations(const String &p_locale);
+	void load_editor_doc_translations(const String &p_locale);
```

Comment:
```suggestion
	void load_editor_translations(const String &p_locale);
	void load_doc_translations(const String &p_locale);
```

To match the header names (especially if moved to an `editor/` file then there's no risk of confusion with project documentation).